tests/ast/data/Session.call.test

```diff
@@ -13,7 +13,7 @@ df2 = session.call("my_sproc", 2, "one", dict(), False)
 
 ## EXPECTED UNPARSER OUTPUT
 
-my_sproc_sp = sproc(my_sproc, name="my_sproc", replace=True, comment="The parameters are useless.")
+my_sproc_sp = sproc("my_sproc", return_type=StringType(), input_types=[LongType(), StringType(), MapType(StringType(), StringType(), structured=False), BooleanType()], name="my_sproc", replace=True, comment="The parameters are useless.", _registered_object_name="\"MOCK_DATABASE\".\"MOCK_SCHEMA\".\"MY_SPROC\"")
```

Comment:
if we have two sprocs, would they both use the same registered object name? That doesn't sound right...

Prob. we'd need to:
1.) not print _registered_object_name here
2) Fix local testing to use unique names (prob. preferred)

Reply:
No, there is logic to generate the (registered) object name, it could be random or based on the "name" of the sproc.  So 2 different sprocs wouldn't normally end up with the same registered object name unless by random chance or they were given the same name.  Either way, the behavior would be same as if was entirely executed on the client.  Let me know if i misunderstood anything here.